x-pack/plugins/apm/public/components/shared/time_comparison/index.tsx

```diff
@@ -41,13 +41,13 @@ export function TimeComparison() {
   const { start, end } = useTimeRange({ rangeFrom, rangeTo });
 
   const {
-    urlParams: { comparisonEnabled, comparisonType },
+    urlParams: { comparisonEnabled, offset },
   } = useLegacyUrlParams();
 
   const comparisonOptions = getComparisonOptions({ start, end });
 
   // Sets default values
-  if (comparisonEnabled === undefined || comparisonType === undefined) {
+  if (comparisonEnabled === undefined || offset === undefined) {
     urlHelpers.replace(history, {
```

Comment:
do we still need this redirect?

Reply:
Removed it in https://github.com/elastic/kibana/pull/127812/commits/f0c8d72ef433c74b468617d6f0fb144d7af65f82 as we now redirect from `RedirectWithOffset`